rasa/core/policies/rule_policy.py

```diff
@@ -153,6 +153,26 @@ def predict_action_probabilities(
         last_action_was_rejection = active_form_name and tracker.events[
             -1
         ] == ActionExecutionRejected(active_form_name)
+        should_predict_form = (
```

Comment:
reverted these changes due to some failing tests. Will drop it later again

Reply:
kept the revert for laziness as I don't want to adapt the tests now 😆

Reply:
I'm updating this

Reply:
yep. Reverted my changes in this file so that it's the same as in the current `rules` branch